Composer/packages/client/src/pages/botProject/BotProjectInfo.tsx

```diff
@@ -24,14 +24,19 @@ const valueStyle = css`
 `;
 
 const headerStyle = css`
-  font-size: 16px;
+  font-size: 19px;
```

Comment:
19px seems weird. We should try to stick to multiples of 4. 20px?

Reply:
Handled